keystorage/src/main/java/tech/pegasys/web3signer/keystorage/azure/AzureKeyVault.java

```diff
@@ -103,6 +115,43 @@ public CryptographyClient fetchKey(final String keyName, final String keyVersion
         .buildClient();
   }
 
+  public HttpRequest getRemoteSigningHttpRequest(
+      final byte[] data,
+      final SignatureAlgorithm signingAlgo,
+      final String vaultName,
+      final String azureKeyName,
+      final String azureKeyVersion) {
+
+    final String apiVersion = KeyServiceVersion.getLatest().getVersion();
+
+    final JsonObject jsonBody = new JsonObject();
+    jsonBody.put("alg", signingAlgo);
+    jsonBody.put("value", Bytes.of(data).toBase64String());
+
+    final String uriString =
+        constructAzureSignApiUri(vaultName, azureKeyName, azureKeyVersion, apiVersion);
+    final URI uri = URI.create(uriString);
+
+    final HttpRequest httpRequest =
```

Comment:
nit: local variable isn't necessary we can just return the request

Reply:
done